src/index/gpu_raft/gpu_raft_ivf_pq_config.h

```diff
@@ -14,62 +14,47 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#ifndef GPU_RAFT_IVF_PQ_CONFIG_H
+#define GPU_RAFT_IVF_PQ_CONFIG_H
 
-#ifndef IVF_RAFT_CONFIG_H
-#define IVF_RAFT_CONFIG_H
-
-#include "index/ivf/ivf_config.h"
 #include "knowhere/config.h"
+#include "index/ivf/ivf_config.h"
+#include "common/raft/proto/raft_index_kind.hpp"
+#include "common/raft/integration/raft_knowhere_config.hpp"
 
 namespace knowhere {
 
-class RaftIvfFlatConfig : public IvfFlatConfig {
- public:
+struct GpuRaftIvfPqConfig : public IvfPqConfig {
+    CFG_FLOAT refine_ratio;
```

Comment:
This newly-introduced parameter allows additional refinement after an initial selection of candidates from an index search.